#! /bin/sh /usr/share/dpatch/dpatch-run
## desktop_icon.dpatch by Samuel Mimram <smimram@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Correct path in .desktop files.

@DPATCH@
diff -urNad linphone-1.2.0~/share/linphone_applet.desktop linphone-1.2.0/share/linphone_applet.desktop
--- linphone-1.2.0~/share/linphone_applet.desktop	2002-02-08 08:54:01.000000000 +0100
+++ linphone-1.2.0/share/linphone_applet.desktop	2006-01-09 00:50:09.000000000 +0100
@@ -5,5 +5,5 @@
 Comment[de]=Linphone ist ein web-phone.
 Type=Application
 Exec=linphone --applet --activate-goad-server=linphone_applet
-Icon=linphone/linphone2.png
-Terminal=0
\ No newline at end of file
+Icon=/usr/share/pixmaps/linphone/linphone2.png
+Terminal=0
diff -urNad linphone-1.2.0~/share/linphone.desktop linphone-1.2.0/share/linphone.desktop
--- linphone-1.2.0~/share/linphone.desktop	2003-02-03 17:52:01.000000000 +0100
+++ linphone-1.2.0/share/linphone.desktop	2006-01-09 00:49:56.000000000 +0100
@@ -5,6 +5,6 @@
 Comment[de]=Linphone ist ein web-phone.
 Type=Application
 Exec=linphone
-Icon=linphone/linphone2.png
+Icon=/usr/share/pixmaps/linphone/linphone2.png
 Terminal=0
 Categories=Application;Network;
